tools/sof-tplgreader.py

```diff
@@ -228,8 +228,73 @@ def func_getPipeline(tplgObj, tplgName, sdcard_id, sort):
     parser.add_argument('-o', '--sort', action='store_true', help="sort pipeline by id for the same tplg")
     parser.add_argument('--version', action='version', version='%(prog)s 1.0')
 
+    # command line arguments for topology graph
+    parser.add_argument('-G', '--graph', action='store_true', help="enable topology graph feature")
+    parser.add_argument('-D', '--directory', type=str, default=".", help="output directory for generated graph")
```

Comment:
If it parses only one file at a time why we need -D option? filename can have directory path also.
if -D option is present, how about search all *.tplg and generate graph file?
./sof-tplgreader.py -G -F png -D PATH

Reply:
The -D here controls where the generated graph will be stored(mostly, should be the log folder, and graph will upload to jenkins as well as log files). It's not the directory of topology.

Reply:
I see then -O, --output is better option.